htop.1.in

```diff
@@ -350,7 +350,8 @@ Currently unsupported (always displays '-').
 .LP 
 By default htop reads its configuration from the XDG-compliant path
 ~/.config/htop/htoprc -- the configuration file is overwritten by htop's
-in-program Setup configuration, so it should not be hand-edited.
+in-program Setup configuration, so it should not be hand-edited. The system
+administrator can overwrite defaults by placing a file @sysconfdir@/htoprc.
```

Comment:
This is incorrect. The system defaults are only loaded if the user defaults are not loaded.